cluster/addons/ip-masq-agent/ip-masq-agent.yaml

```diff
@@ -36,3 +33,9 @@ spec:
               # The daemon looks for its config in a YAML file at /etc/config/ip-masq-agent
               - key: config
                 path: ip-masq-agent
+      tolerations:
+        - key: "CriticalAddonsOnly"
+          operator: "Exists"
+        - key: "dedicated"
+          value: "master"
+          effect: "NoSchedule"
```

Comment:
Lets remove the master part as suggested by Marek, but lets leave the CriticalAddonsOnly please.